lib/routes/display_name/post.js

```diff
@@ -15,13 +15,20 @@ const EMPTY = Object.create(null);
 //   \u007F         - ascii DEL character
 //   \u0080-\u009F  - C1 (ansi escape) control characters
 //   \u2028-\u2029  - unicode line/paragraph separator
-//   \uD800-\uDFFF  - non-BMP surrogate pairs
```

Comment:
If we're removing it from the comment, should it also be removed from the definition of `ALLOWED_DISPLAY_NAME_CHARS`?